.circleci/config.yml

```
@@ -26,11 +26,24 @@ jobs:
   hive-optimism-tests:
     machine:
       image: ubuntu-2004:202201-02
+      docker_layer_caching: true
     steps:
       - checkout
       - attach_workspace: {at: "/tmp/build"}
       - run:
-          command: "/tmp/build/hive -sim=\"optimism/*\" -sim.loglevel=5 -client=go-ethereum,op-geth,op-proposer,op-batcher,op-node,op-contracts |& tee /tmp/build/hive.log"
+          command: |
+            /tmp/build/hive \
+            -sim='optimism/*' \
```

```suggestion
            -sim=optimism \
```

nit: it's regex by default